### ISSUE TITLE
`This-is-awesome` I would like to join

### ISSUE THREAD

**Issue by barahonajm** [dotnet-foundation/wg-education#18]
Hello, thanks for putting you efforts on creating this, education is definitely something awesome to invest on.

Sorry if this was not the place to apply but the link did not work for me, to keep this short (and not bore you 😊) I would like to be part of the committee if possible and help to make this a reality, I am currently working as a Software Engineer, I already have some experience imparting courses to employees and students and also I'm being contributing to the ASP.NET Core repo from time to time.

I like education so I am sure I will be able to contribute and also at some point I would like to take what we might build and make it available to the Spanish speakers community (since I am one).

Thanks in advance.

**Comment by bjablonsky** [comment]
Awesome. 

It looks like you aren't a member of the DNF. Unfortunately, I can't add you to the DNF org's education team on GitHub. Please see this announcement for details about the meeting: https://github.com/orgs/dotnet-foundation/teams/education/discussions/8

**Comment by huunab** [comment]
@bjablonsky Yes I was thinking the same, I have already applied for being part of DNF a couple of days ago (crossing fingers), thanks for you response, it looks like I cannot enter to the link you have provided me as probably discussion are not open for public .

**Comment by bjablonsky** [comment]
Ah. Sorry. I didn't realize the link I sent was member's only. The membership process can take some time. In the meantime, can you email me so I can get you added to the meeting invite: dotnetfoundation -at- brianjablonsky -dot- com

**Comment by bjablonsky** [comment]
Just wanted to follow up. Did you ever get your DNF application processed? Looks like you have a meeting invite. Wanted to double check, did you get both the 2nd and 4th Wednesday meeting invites?
